backend/compact-connect/stacks/persistent_stack/staff_users.py

```diff
@@ -33,7 +34,7 @@ def __init__(
             removal_policy=removal_policy,
             **kwargs
         )
-        stack: AppStack = AppStack.of(self)
+        stack: ps.PersistentStack = ps.PersistentStack.of(self)
```

Comment:
What changed here? Why was `AppStack` previously correct?

Reply:
`AppStack` is the base class of `PersistentStack`. We're using a more specific attribute (`PersistentStack.alarm_topic`), so I'm including a more specific type hint.